app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass

```

Comment:
It's a good idea to provide a custom error message for the `NotVaccinatedError` class. This will make your error handling more descriptive and easier to understand. You can do this by overriding the `__init__` method in your class. Here's an example:

```python
class NotVaccinatedError(VaccineError):
    def __init__(self, message="Not vaccinated error occurred."):
        super().__init__(message)
```